Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -86,6 +86,39 @@ open class ChatMessageListVC: _ViewController,
         return (!listView.isLastCellFullyVisible && isMoreContentThanOnePage) || dataSource?.isFirstPageLoaded == false
     }
 
+    /// A button to scroll the collection view to the first unread message.
```

Comment:
Are we not using a tableview? 🤔

Reply:
I think I copy pasted it from some other value 🤔 Corrected it now